vignettes/LPWC.Rmd

```diff
@@ -80,9 +80,10 @@ library(LPWC)
 
 ## Required input
 
-**Data**:  Input to LPWC is a matrix.
+**Data**:  Input to LPWC is a matrix/data.frame.
 We assume here that it is a gene expression matrix, but other data types are suitable as well.
-The expression matrix should be n -by- p where n is the number of genes (in rows) and p is the number of timepoints.
+The expression matrix should be n -by- p where n is the number of genes (in rows) and p is the number of timepoints. The object names can be specified in the row names of ythe data frame 
+using \code{rownames}.
```

Comment:
Does the syntax `\code{}` work in vignettes as well?  If it follows Markdown syntax it won't, so I may change this.